### PR TITLE
(port) New Pet, Frank the Turtle!

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr: https://github.com/BeeStation/BeeStation-Hornet/pull/1413
"Adds an adorable new pal to the mining base, Frank the Turtle (Lovingly referred to as Yeeslow by some members of the Crew). Don't let his adorable appearance fool you, Frank is built like a tank and can take a lot of punishment. Don't expect him to come to the rescue though, as he likes to get down low and hide in his shell whenever he's in danger."

he is found in lavaland base

#### Changelog

:cl:  
rscadd: Added frank  the turtle
/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
gotta collect em all station pets!

File: code/modules/mob/living/simple_animal/friendly/turtle.dm [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 1, /obj/item/clothing/head/franks_hat = 1)
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm = "kicks"
+	mob_biotypes = list(MOB_ORGANIC, MOB_BEAST)
+	gold_core_spawnable = FRIENDLY_SPAWN
+	melee_damage_lower = 18
+	melee_damage_upper = 18
+	health = 2500
+	maxHealth = 2500
```
Comment (Hopekz) [inline code/modules/mob/living/simple_animal/friendly/turtle.dm]: wut

**Comment by Partheo** [comment]
Personally not a fan of more pets, I think there should be 2-3 max.

**Review by Partheo — APPROVED** [review]
Sprites fine

**Review by patpol4 — CHANGES_REQUESTED** [review]
Not a fan of pet oversaturation but i can't help myself when i see a cowboy hat on a turtle.Make this compile please.

**Review by patpol4 — APPROVED** [review]
Map technically fine, whoever presses the merge button think about if we need more pets.

**Pull request merged**